### PR TITLE
Task nav

### PR DESCRIPTION
**Pull request by ajhswan**
Code for the nav bar and the header/hero section of Portfolio site. Includes media queries to make site responsive and includes all image assets (header image and burger menu icon)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+				<li>About Me</li>
+				<li>Contact</li>
+			</ul>
+			<img src='list-menu.png'/>
+		</nav>
+		
+		<header>
+			<div class='content'>
+				<h1>Andrew Swan</h1>
+				<h2>fullStack_developer</h1>
```
Comment (mathewjmartin) [inline index.html]: Header tags need to be the same

File: index.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+				<h3>executing ideas of the future</h2>
```
Comment (mathewjmartin) [inline index.html]: Need same header tags

File: index.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Portfolio Site</title>
+		<link rel='stylesheet' type='text/css' href='styles.css'>
+		<link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:300&display=swap" rel="stylesheet">
+		<link href="https://fonts.googleapis.com/css?family=Work+Sans:900&display=swap" rel="stylesheet">
```
Comment (mathewjmartin) [inline index.html]: Normalise CSS file?

File: index.html (hunk 1 above)
Comment (mathewjmartin) [inline index.html]: When shrinking the browser window to mobile size, this text gets lost

**Review by mathewjmartin — COMMENTED** [review]
Checked with a few comments.

File: index.html (hunk 3 above)
Comment (ajhswan) [reply index.html]: IE browser check seemed to be fine and I'm not updating any strange properties. I might add at the end of the build

File: index.html (hunk 1 above)
Comment (ajhswan) [reply index.html]: Thanks - done

Comment (ajhswan) [reply index.html]: seems fine to me - may be fixed once the h2 and h3 tags are updated

**Pull request merged**